src/buildMathML.js

```diff
@@ -167,6 +168,20 @@ groupTypes.color = function(group, options) {
 };
 
 groupTypes.supsub = function(group, options) {
+    // Is the inner group a relevant horizonal brace?
+    let isBrace = false;
+    let isOver;
+    let isSup;
+    if (group.value.base) {
+        if (group.value.base.value.type === "horizBrace") {
+            isSup = (group.value.sup ? true : false);
+            if (isSup === group.value.base.value.isOver) {
+                isBrace = true;
```

Comment:
Well, I used an attribute name that is confusing. I was worried about that. 

Back in `functions.js`, where I handle the functions `\overbrace`, `\underbrace`, `\overbracket`, and `\underbracket`, I  give the type attribute `horizBrace` to all four of them. I should have named that type attribute `horizBraceOrBracket`. Unless you can think of something less verbose.